modules/dot-prefix-validation/build.gradle

```diff
@@ -26,4 +26,5 @@ tasks.named('yamlRestTest') {
 
 tasks.named('yamlRestCompatTest') {
   usesDefaultDistribution()
+  systemProperty 'es.queryable_built_in_roles_enabled', 'false'
```

Comment:
Only disabling it for `yamlRestCompatTest` which fail due to a warning header because the yml test now deletes `.security` index. I will be handling this in a followup after backporting the changes to 8.18.0 branch.

Reply:
They are, but they did not account for `.security` index. The fix is to expect the warning header, or adjust the test not to fetch/delete `.security` index. But, it's not as easy as that, since we also run `yamlRestCompatTest`, which execute the old tests (from a previous version) against the new cluster. So, even if I adjust test here and they pass, the same older version of this test will be executed as part of `yamlRestCompatTest` and fail since it has no updated assertions. 
The solutions are to rewrite the test completely on all branches, or to have some conditional execution depending on the cluster_features (not sure if this would work completely).